frontend/cypress/integration/common/services.ts

```diff
@@ -68,6 +68,12 @@ When('user filters for label {string}', (label: string) => {
   cy.get('input[aria-label="filter_input_label_key"]').type(`${label}{enter}`);
 });
 
+
+When('user applies annotations', () => {
```

Comment:
I would specify in the title what type of annotations I am applying (for example: `user applies kiali api annotations`)